arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+
+In the introductory section, we outlined three primary configurations required for setting up and managing an Orbit chain. These configurations play a crucial role in ensuring the efficient operation of your chain. Let's delve into each type:
+
+## 1. Parent Chain Configuration
+Configuring the parent chain is an essential initial step in setting up your Orbit chain. Most of these configurations are specified during the setup phase. Detailed instructions can be found in the [Rollup Deployment Parameters](deploying-rollup-chain-with-sdk#rollup-deployment-parameter) section of the rollup deployment guide. 
+
+After the initial setup, the chain owner can modify configurations as needed. For instance, the validator set can be updated by invoking the [`setValidKeyset`](https://github.com/OffchainLabs/nitro-contracts/blob/90037b996509312ef1addb3f9352457b8a99d6a6/src/bridge/SequencerInbox.sol#L751) function with a new set of validators. This adaptability facilitates continuous optimization and management of the chain.
+
+## 2. Node/Chain Configuration
+This category includes settings adjustable within the `nodeConfig.json` file, directly impacting the operation of the chain's nodes, including special nodes like validators and sequencers. These settings are vital for tailoring the node's functionality to specific requirements or performance criteria. The chain owner can modify these configurations during the node config generation process, ensuring that each node operates with the desired settings from the start. For more information, refer to the [Node Configuration Preparation](preparing-node-config-with-sdk.md) documentation.
+
+## 3. Child Chain Parameter Configuration
+The final configuration type involves setting parameters on the child chain. This level of configuration is primarily achieved through the [ArbOwner precompile](https://github.com/OffchainLabs/nitro-contracts/blob/main/src/precompiles/ArbOwner.sol) on the child chain. These configurations are typically applied post-chain initialization and after the deployment of the token bridge. This guide will help you configure child chain parameters using the Orbit SDK, providing insights into effective management and optimization strategies for your child chain operations.
+
+The child chain configuration can be performed after the chain initialization. These parameters are configurable via setter functions in the [ArbOwner precompile](https://github.com/OffchainLabs/nitro-contracts/blob/main/src/precompiles/ArbOwner.sol). Additionally, there are various getter functions in the ArbOwner precompile that you can use to read the current configuration. Below, we explain several methods in the ArbOwner precompile that you can use to configure the parameters or read their current state.
+
+### Setter Functions
+You can use these setter functions to configure the child chain parameters:
```

Comment:
This reads similar to reference content; to maintain consistency, we can reformat as a table. As a future optimization / separate PR, we can relocate this into a dedicated reference document (primarily "tables of things"), and then either link to it here, or partialize it and include it in both.

```markdown
| Method                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `addChainOwner`       | Allows you to add a new chain owner to your Orbit chain.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
| `removeChainOwner`    | Enables you to remove an existing owner from the list of chain owners.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
| `setMinimumL2BaseFee` | Sets the minimum base fee on the child chain. The minimum base fee is the lowest amount that the base fee on the child chain can ever be. For example, the current minimum base fee on Arbitrum One is 0.1 gwei, and on Arbitrum Nova, it is 0.01 gwei.                                                                                                                                                                                                                                                                                                                                |
| `setSpeedLimit`       | The fee mechanism on the Arbitrum Nitro stack differs from the Ethereum blockchain. The Nitro stack has a parameter called the speed limit, which targets the number of gas consumed on the child chain per second. If the amount of gas per second exceeds this pre-specified amount, the base fee on the child chain will increase, and vice versa. The current speed limit on Arbitrum One is 7 million gas per second, meaning if the Arbitrum One chain consumes more than 7 million gas per second, its base fee will increase. For more information on the speed limit, please refer to this [document](https://docs.arbitrum.io/inside-arbitrum-nitro/#the-speed-limit). |
| `setInfraFeeAccount`  | Sets the infrastructure fee account address, which receives all fees collected on the child chain. It is meant to receive the minimum base fee, with any amount above that going to the network fee account.                                                                                                                                                                                                                                                                                                                                                                           |
| `setNetworkFeeAccount`| Sets the network fee account address. As mentioned, this address collects all fees above the base fee. Note that if you set this amount to the 0 address on your chain, all fees will be deposited into the infrastructure fee account.                                                                                                                                                                                                                                                                                                                                                 |
| `scheduleArbOSUpgrade`| If you plan to upgrade the <a data-quicklook-from="arbos">ArbOS</a> version of your chain, this method can help you schedule the upgrade. For a complete guide on this matter, please refer to the explanation of the [arbos upgrade](../../how-tos/arbos-upgrade.md).                                                                                                                                                                                                                                                                                                                   |
| `setChainConfig`      | We discussed the chainConfig in the [Rollup deployment guide](deploying-rollup-chain-with-sdk.md#chain-config-parameter) in detail. If you wish to change any field of the `chainConfig`, you need to use this method on the child chain.                                                                                                                                                                                                                                                                                                                                               |
````